convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -63,3 +79,81 @@ def run(self):
 
         repo.backup_yum_repos()
         repo.backup_varsdir()
+
+
+class BackupPackageFiles(actions.Action):
+    id = "BACKUP_PACKAGE_FILES"
+
+    def run(self):
+        """Backup changed package files"""
+        loggerinst.task("Prepare: Backup package files")
+
+        super(BackupPackageFiles, self).run()
+
+        package_files_changes = self._parse_rpm_va()
+
+        for file in package_files_changes:
+            if file["status"] == "missing":
+                pass  # no action needed now
+            elif re.search(r"[5]", file["status"]):
```

Comment:
My motivation there to use the regex was the ability to easily add other symbols. But who knows if it will be needed and for now, this is fine. I'll change it